sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java

```diff
@@ -917,4 +920,25 @@ public String javascriptGetElementContent(String selector) {
     String content = javascriptExecutor.executeScript(script).toString();
     return content;
   }
+
+  public String returnURL() {
+    return baseSteps.getDriver().getCurrentUrl();
+  }
+
+  public void switchToOtherWindow() {
+    String parent = baseSteps.getDriver().getWindowHandle();
+    Set<String> S = baseSteps.getDriver().getWindowHandles();
```

Comment:
add also a check to verify that we have  available tabs to switch to, so in case of failure we can see in logs